sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment.py

```diff
@@ -573,3 +574,93 @@ def callback(pipeline_response, deserialized, _):
             cls=callback
         )
         assert res == "cls result"
+
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    def test_aspect_based_sentiment_analysis(self, client):
+        documents = [
+            "It has a sleek premium aluminum design that makes it beautiful to look at."
+        ]
+
+        document = client.analyze_sentiment(documents=documents, show_opinion_mining=True)[0]
+
+        for sentence in document.sentences:
+            for mined_opinion in sentence.mined_opinions:
+                aspect = mined_opinion.aspect
+                self.assertEqual('design', aspect.text)
+                self.assertEqual('positive', aspect.sentiment)
+                self.assertIsNotNone(aspect.confidence_scores.positive)
+                self.assertEqual(0.0, aspect.confidence_scores.neutral)
+                self.assertIsNotNone(aspect.confidence_scores.negative)
```

Comment:
we could add an extra check where all the confidence scores need to add up to 1. That way we know for sure that not all of them are `0`

Reply:
For more context, in .NET I was doing the exact checks and I missed a bug in the deserialization, so all scores always ended up in `0.0`.
The adds up to check has caught some of those failures

Reply:
that's fair. i've created a separate issue to add this, since there are more instances in our test suite where i'd like this added: https://github.com/Azure/azure-sdk-for-python/issues/12833